lib/helpers/export-results.js

```diff
@@ -0,0 +1,83 @@
+
+const fs = require('fs');
+
+/**
+ * @param {Object} opts
+ * @param {String} opts.destFolder  - folder with annotated files
+ * @param {String} opts.resultsFilepath  - a path where should be saved result-file
+ * @returns {} result-file is saved
+ */
+
+/**
+ * @typedef {String} epoch - timestamp, shows when result was generated
+ * @example "2019-11-06T16:32:27.712Z"
+ */
+
+/**
+ * @typedef {Array.<String>} filenames - a list of filenames
+ * @example ["111111_23_24-2-1570481465391-person-5c468.gif", ..]
+ */
+/**
+ * @typedef {String} thumbnail - a name of thumbnail
```

Comment:
new defined types must start with an uppercase

Reply:
I do not think it makes sense to create a specific `Thumbnail` type, `Thumbnail` is excatly the same type as `Filename`